data/users/externalldap.py

```diff
@@ -33,14 +35,15 @@ def __init__(
         self._timeout = timeout
         self._network_timeout = network_timeout
 
-    def get_connection(self):
+    def get_connection(self, referral=1):
```

Comment:
I don't think following referrals should be on by default, since the assumption that the same credentials will work on a separate server is kinda broken.